.vib/common/goss/templates/check-nginx-libphp.yaml

```
@@ -0,0 +1,12 @@
+file:
+  /opt/bitnami/nginx/conf/bitnami/php-fpm.conf:
+    exists: true
+    filetype: file
+    contains:
+      - "index index.html index.htm index.php;"
+      - "location ~ \\.php$ {"
+      - "fastcgi_read_timeout 300;"
+      - "fastcgi_pass   unix:/opt/bitnami/php/var/run/www.sock;"
+      - "fastcgi_index  index.php;"
+      - "fastcgi_param  SCRIPT_FILENAME $request_filename;"
+      - "include        fastcgi_params;"
```

This is subjective, but I don't think there is need to check line by line. Instead, I would go with the more relevant content 😁 

```suggestion
      - "location ~ \\.php$ {"
      - "fastcgi_pass   unix:/opt/bitnami/php/var/run/www.sock;"
      - "include        fastcgi_params;"
```

Again this is not wrong, but something that came to my mind